build/config/PublishData.json

```diff
@@ -48,6 +48,13 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn-nonnull/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn-nonnull/vsix/upload" ],
             "channels": [ "nonnull" ]
+        },
+        "features/dataflow": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.8.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
```

Comment:
Historically we've provided a separate feed, as it's not uncommon for partners to grab packages from our main feed without realizing it. Can we create a new feed? You should just be able to go to dotnet.myget.org and make one yourself.

Reply:
These packages have a different pre-release tag hence same feed should be fine.

Reply:
@jasonmalinowski Any further concerns here?